src/clj/test/cards/upgrades.clj

```diff
@@ -633,6 +633,54 @@
       (is (= 5 (:credit (get-runner))) "Runner was not charged 5cr")
       (is (= 1 (count (:scored (get-runner)))) "1 scored agenda"))))
 
+(deftest ruhr-valley
+  "Ruhr Valley - As an additional cost to make a run on this server, the Runner must spend a click."
```

Comment:
Can wrap the testing statements in `(testing "description-string" ...)` as well.